models/src/anemoi/models/schemas/models.py

```diff
@@ -188,7 +191,10 @@ class BaseModelSchema(PydanticBaseModel):
         ...,
         discriminator="target_",
     )
-    "GNN decoder schema."
+    residual: Union[SkipConnectionSchema, TruncationMapperSchema, NoConnectionSchema] = Field(
```

Comment:
The use of a default factory is nice for preventing a change to configs, what do you think about adding a commented out line in the configuration file for models `#residual: SkipConnectionSchema` so that people can see where to change the type of residual connection?